WordPress/Classes/Utility/Editor/GutenbergSettings.swift

```diff
@@ -157,6 +158,15 @@ class GutenbergSettings {
         database.set(true, forKey: Key.enabledOnce(for: blog))
     }
 
+    /// True if it should show the tooltip for the starter page templates picker
+    func getStarterPageTemplatesTooltipShown() -> Bool {
+        return database.bool(forKey: Key.starterPageTemplatesTooltipShown)
+    }
+
+    func setStarterPageTemplatesTooltipShown(_ tooltipShown: Bool) {
+        database.set(tooltipShown, forKey: Key.starterPageTemplatesTooltipShown)
+    }
+
```

Comment:
This suggestion is a minor style change, in Swift, we usually omit the `get` keyword for methods like these such as in:
https://github.com/wordpress-mobile/WordPress-iOS/blob/617a7a3092c8184bc14bf9ef572297a58ab021f0/WordPress/Classes/Utility/Editor/GutenbergSettings.swift#L96-L102

So maybe something like this would be more inline with the pattern you have if you wanted to keep it as function calls:
```suggestion
    /// True if it should show the tooltip for the starter page templates picker
    func starterPageTemplatesTooltipShown() -> Bool {
        return database.bool(forKey: Key.starterPageTemplatesTooltipShown)
    }

    func setStarterPageTemplatesTooltipShown(_ tooltipShown: Bool) {
        database.set(tooltipShown, forKey: Key.starterPageTemplatesTooltipShown)
    }

```

---

Another option that is more "Swift-like" would be to write it as  computed property:
```suggestion
    /// True if it should show the tooltip for the starter page templates picker
    var starterPageTemplatesTooltipShown: Bool {
        get {
            database.bool(forKey: Key.starterPageTemplatesTooltipShown)
        }

        set {
            database.set(newValue, forKey: Key.starterPageTemplatesTooltipShown)
        }
    }

```

You could then access it with `GutenbergSettings().starterPageTemplatesTooltipShown` and `GutenbergSettings().starterPageTemplatesTooltipShown = tooltipShown `

Reply:
TIL! Thank you so much for these suggestions Chip! I'll update the code 🙌